indico/modules/events/papers/controllers/management.py

```diff
@@ -123,6 +125,23 @@ def _process(self):
                                               form=form)
 
 
+class RHManageContact(RHManagePapersBase):
+    """Allow to send emails to members of reviewing team"""
+
+    def _process(self):
+        paper_persons_dict = {}
+        for p in self.event.acl_entries:
+            paper_persons_dict[p]['judge'] = p.has_management_role('paper_judge', explicit=True)
```

Comment:
Did you test this after changing it? It'll fail since it's not a defaultdict anymore. Also, you are adding people to the `paper_persons_dict` who are in the event ACL but not related to the papers module.

Check whether the user has the roles first and only then set `paper_persons_dict[p] = {...}` if he has at least one role.